code/library/src/main/kotlin/org/fbme/lib/st/expressions/UnaryOperation.kt

```diff
@@ -1,7 +1,7 @@
 package org.fbme.lib.st.expressions
 
-enum class UnaryOperation(val alias: String, val isSpaced: Boolean) {
-    NOT("NOT", true), NEG("-", false);
+enum class UnaryOperation(val alias: String, val luaAlias: String, val isSpaced: Boolean) {
```

Comment:
Same here, I would make a separate function in `4diac-integration` to map st operations to lua operations

Reply:
fixed